elk/default_config.json

```diff
@@ -11,14 +11,13 @@
     "piqa",
     "story-cloze"
   ],
-  "models": [
-    "t5-11b",
-    "unifiedqa-t5-11b",
-    "T0pp",
-    "gpt-j-6B",
-    "roberta-large-mnli",
-    "deberta-v2-xxlarge-mnli"
-  ],
+  "model_shortcuts" : {
```

Comment:
As it is right now, it will lead to have slashes in the directory name. Not sure if this is desirable. Maybe we should avoid that, or replace the string when creating the folders.

Reply:
Hmmm it may actually be fine if we're consistent about it? Like you just end up creating an `EleutherAI` parent folder. But idk.